### ISSUE TITLE
New packages with additional dlls

### ISSUE THREAD

**Issue by LoicBaumann** [mellinoe/ImGui.NET#279]
@mellinoe it would be awesome if a new package could be published along with all the DLL to make ImPlot/Gizmo and others embedded with their respected cimgui dlls.

I've followed the instructions to build the cimgui dll locally, but ImPlot is crashing when I create a Plot (the context was created correctly, no crash). :(

**Comment by mbwilding** [comment]
Yeah same issue, they changed the params that the BeginPlot takes in, so the binary you're using is mismatched. Would be nice if they were included per release as you said.
